src/Symfony/Bundle/TwigBundle/Command/DebugCommand.php

```diff
@@ -57,8 +58,9 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
```

Comment:
you should not replace the ``$output`` variable which will be passed to the parent IMO